templates/scenarios/ts/link-unfurling/package.json.tpl

```diff
@@ -0,0 +1,36 @@
+{
+    "name": "{{appName}}",
+    "version": "1.0.0",
+    "description": "Microsoft Teams Toolkit m365 message extension sample",
+    "engines": {
+        "node": "16 || 18"
+    },
+    "author": "Microsoft",
+    "license": "MIT",
+    "main": "./lib/index.js",
+    "scripts": {
+        "dev:teamsfx": "env-cmd --silent -f .localConfigs npm run dev",
+        "dev": "nodemon --exec node --inspect=9239 --signal SIGINT -r ts-node/register ./index.ts",
+        "build": "tsc --build",
+        "start": "node ./lib/index.js",
+        "test": "echo \"Error: no test specified\" && exit 1",
+        "watch": "nodemon --exec \"npm run start\""
+    },
+    "repository": {
+        "type": "git",
+        "url": "https://github.com"
+    },
+    "dependencies": {
+        "botbuilder": "^4.18.0",
+        "restify": "^10.0.0"
+    },
+    "devDependencies": {
+        "@types/restify": "^8.5.5",
+        "@types/node": "^14.0.0",
```

Comment:
We already use node 18 by default, why still use @types/node 14 here?

Reply:
Good catch. Let me update.